WordPress/Classes/Stores/NoticeStore.swift

```diff
@@ -34,6 +34,11 @@ struct Notice {
     ///
     let cancelTitle: String?
 
+    /// An optional value that can be used as a reference by consumers.
+    ///
+    /// This is not used in the Notice system at all.
+    let tag: String?
```

Comment:
Really minor comment, but it might be nice to make this tag a `typealias` for String. If you go with my other suggestion to make the tag an argument for `dismissNotice(tagged:)` I think it's slightly nicer to have it be `dismissNotice(tagged tag: NoticeTag)` than `dismissNotice(tagged tag: String)`, but I'm really being picky here 😆

Reply:
@nheagy I have not used `typealias` a lot. Would you mind sharing more why that would be beneficial in here? I'm afraid I don't understand. 😬

Reply:
Here's a decent overview: http://artsy.github.io/blog/2016/06/24/typealias-for-great-good/